templates/appliance/governance.html

```diff
@@ -0,0 +1,133 @@
+{% extends "appliance/_base-appliance.html" %}
+
+{% block title %}Ubuntu Appliances Governance{% endblock %}
+
+{% block meta_description %}Governance ensures that understandable tensions are resolved productively.{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1DpU0AzAJ4gBa_au4gELBUD3vLz8l-FnAwFdERBHlvwc/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip--suru-topped">
+  <div class="row">
+    <div class="col-8">
+      <h1>Governance</h1>
+      <p>A community forms around a shared interest, but a healthy and large community will inevitably have to manage divergent or competing goals. For example, in this community, we have appliance software from multiple vendors. It is in all of our interests to deliver both the highest quality experience for individual appliances and multi-vendor interoperability and substitution.</p>
+      <p>Governance ensures that understandable tensions are resolved productively. In general, we lean towards ownership over a committee, because doing the work earns the right to decide how best it should be done. Where there are significant questions that touch on multiple contributors, we encourage them to work out a constructive common ground, and we offer an escalation path to resolve matters that do not reach quick consensus.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-8">
+      <h2>Inclusion process</h2>
+      <p>To make your application into an Ubuntu appliance you need to do the following things:</p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked"><a href="https://snapcraft.io/" class="p-link--external">Package your software as a snap</a></li>
+        <li class="p-list__item is-ticked">Publish your snap in the store</li>
+        <li class="p-list__item is-ticked">Test your software on certified hardware</li>
+        <li class="p-list__item is-ticked">Get in touch to discuss your launch</li>
+        <p>At any stage of the process, publishers can get-in-touch regarding any questions or problems through the <a href="https://forum.snapcraft.io/" class="p-link--external">snapcraft forum</a>. Here people will be able to guide you in packaging and publishing your software. Similarly, if you are unable to test your software on all of the hardware you want the forum is where you will find people to do testing for you. When you get in touch we will qualify your tests and verify the quality of your snap. After that, it is simply a case of creating your page on our website and planning any marketing you would like to do. Once your appliance is a part of the portfolio, it is up to you to determine whether your appliance will be considered <a href="#certified">&lsquo;certified&rsquo;</a>, <a href="#maintained">&lsquo;maintained&rsquo;</a>, or <a href="#experimental">&lsquo;experimental&rsquo;</a>.</p>
```

Comment:
"get in touch" should be a link that triggers the contact modal (according to the copy doc)